app/ts/components/subcomponents/Spinner.tsx

```diff
@@ -1,11 +1,20 @@
-export function Spinner() {
+export function Spinner({ height } : { height: string }) {
 	return (
 		<svg
-			style = { { height: '1em' } }
+			style = { { height, margin: 'auto'  } }
 			class = 'spinner'
 			viewBox = '0 0 100 100'
 			xmlns = 'http://www.w3.org/2000/svg'>
 				<circle cx = '50' cy = '50' r = '45'/>
 		</svg>
 	)
 }
+
+export function CenterToPageTextSpinner({ text } : { text: string }) {
+	return <main class = 'center-to-page'>
+		<div style = 'display: grid;'>
```

Comment:
```suggestion
		<div style = 'display: grid; place-items: center;'>
```

might be worth ensuring horizontal centering of the spinner in case some wrapper overrides text center